addons/web/static/src/scss/list_view.scss

```diff
@@ -342,6 +342,9 @@
         text-align: center;
         line-height: 30px;
         z-index: 1; // must be over the resize handle
+        @supports (-moz-appearance:none) {
+			padding: 0 15px;
```

Comment:
Weird indent, can you fix it?

Reply:
@rdeodoo i have corrected, please review for me